pages/product_list.html

```diff
@@ -0,0 +1,389 @@
+                </div>
+                <div class="btn_content">Add to Cart</div>
+              </button>
+              <button class="btn btn-outline btn-wrapper">
+                <div class="btn_icon"><i class="fa-solid fa-heart"></i></div>
+                <div class="btn_content">Wishlist</div>
+              </button>
+            </div>
+          </div>
+          <div class="as-card__wishlist">
+            <i class="fa-brands fa-gratipay"></i>
+          </div>
+        </a>
+        <a class="card flex-column as-prod__card">
+          <div class="card__img as-prod__img">
+            <img src="/Assets/Badminton_racket_2.jpeg" alt="Sofa" />
+          </div>
+          <div class="card__info pad-8-16">
+            <p class="card-title">Yonex</p>
+            <p class="card-desc">Astrox 88D Badminton Racket</p>
+          </div>
+          <div class="card__price pad-8-16 d-flex">
+            <div class="card__price--final">Rs:2000</div>
+            <div class="card__price--original"><strike>5500</strike></div>
+            <div class="card__price--offer">65% off</div>
+          </div>
+          <div class="card__footer pad-8-16">
+            <div class="card-footer__div flex-column">
+              <button class="btn btn-primary btn-wrapper">
+                <div class="btn_icon">
+                  <i class="fa-solid fa-cart-shopping"></i>
+                </div>
+                <div class="btn_content">Add to Cart</div>
+              </button>
+              <button class="btn btn-outline btn-wrapper">
+                <div class="btn_icon"><i class="fa-solid fa-heart"></i></div>
+                <div class="btn_content">Wishlist</div>
+              </button>
+            </div>
+          </div>
+          <div class="as-card__wishlist">
+            <i class="fa-brands fa-gratipay"></i>
+          </div>
+        </a>
+
+      </div>
+      
+    </div>
+
+  </main>
+  <footer class="footer flex-justify-sp-bw">
+    <address class="footer__address">
+      <h4 class="footer__h2">Registered Address</h4>
+      <p class="footer__P">Street: Plot No 233, Brodipet, Guntur</p>
+      <p class="footer__P">State: Andhra Pradesh</p>
+      <p class="footer__P">Zip code: 122001</p>
+      <p class="footer__P">Country: India</p>
+    </address>
+    <div>
+      <span>©AccelSports. All rights reserved</span>
+    </div>
+    <div class="footer__social">
+      <span>Keep in touch</span>
+      <ul class="footer_ul flex-justify-sp-bw ul-pad-0">
+        <li class="footer_li">
+          <a href="https://github.com/kalyandindu/Accel-UI"><i class="fab fa-github"></i></a>
+        </li>
+        <li class="footer_li">
+          <a href="https://www.linkedin.com/in/pavan-dindu-a839b6122/"><i class="fab fa-linkedin"></i></a>
+        </li>
+        <li class="footer_li">
+          <a href="https://twitter.com/Pavandindu"><i class="fab fa-twitter"></i></a>
+        </li>
+      </ul>
+    </div>
+  </footer>
+</body>
+</html>
```

Comment:
To avoid this red mark at the end of file, just add a single empty line at the end of html tag.

Reply:
Thanks for suggestion, Have updated in latest commit.